artifacthub-repo.yml

```
@@ -0,0 +1,14 @@
+# Artifact Hub repository metadata file
+#
+# Some settings like the verified publisher flag or the ignored packages won't
+# be applied until the next time the repository is processed. Please keep in
+# mind that the repository won't be processed if it has not changed since the
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+owners: # (optional, used to claim repository ownership)
+  - name: hashicorpreleng
```

```suggestion
  - name: hashicorp
```

Should this be `hashicorp` for simplicity?

---

Yeah removed, only the email needs to match the user.